src/routes/(infoPages)/about/news/shared.server.ts

```diff
@@ -0,0 +1,117 @@
+import { CONTENTFUL_SPACE_ID, CONTENTFUL_DELIVERY_API_TOKEN } from "$env/static/private";
+import getContentfulClient from "$lib/services/contentful";
+import gql from "graphql-tag";
+import { print as printQuery } from "graphql";
+import type { NewsEntriesQuery } from "./$queries.generated";
+import { error } from "@sveltejs/kit";
+import type { PageServerLoad } from "./page/[page]/$types";
+import type { Breadcrumbs } from "$lib/components/Breadcrumbs";
+import {
+  newsEntries as testNewsEntries,
+  pages as testNewsEntryPages,
+} from "./__tests__/newsTestContent";
+import imagePropsFragment from "$lib/fragments/imageProps";
+import entryPropsFragment from "$lib/fragments/entryProps";
+
+const limit = 10;
+
+export const query = gql`
+  # eslint-disable @graphql-eslint/selection-set-depth
+  ${imagePropsFragment}
+  ${entryPropsFragment}
+  query NewsEntries($limit: Int = 10, $skip: Int = 0) {
+    newsCollection(limit: $limit, skip: $skip, order: [publicationDate_DESC]) {
+      total
+      items {
+        sys {
+          id
+        }
+        type
+        title
+        publicationDate
+        body {
+          json
+          links {
+            assets {
+              block {
+                ...ImageProps
+              }
+              hyperlink {
+                ...ImageProps
+              }
+            }
```

Comment:
This block might not be necessary since we should only be showing the first paragraph of text in the snippet. Could use a gut-check here though.

Reply:
Yeah, I'll test out removing this as well as `entries.block` (leaving only `entries.hyperlink`) and as long as all the current snippets look good I think we can move forward with a slightly lighter request. Plus, we'll probably be replacing the first paragraph snippet with the `subhead` later anyway.

Reply:
Okay, looks like `imagePropsFragment` is a bit of a misnomer, since its for all assets, not just images. So if we wanted to link to a PDF, it's going to appear under `assets.hyperlink`. I went ahead and updated it again so that we're only grabbing the `hyperlink` portions for `entries` and `assets`. As a follow-up I'll probably rename `imagePropsFragment` to `assetPropsFragment` to avoid future confusion.